ecosystem_tests/ecosystem_tests_cli/ecosystem_tests.py

```diff
@@ -37,7 +37,8 @@
                         DEFAULT_LICENSE_PATH,
                         DEFAULT_BLUEPRINT_PATH,
                         DEFAULT_CONTAINER_NAME,
-                        DEFAULT_UNINSTALL_ON_SUCCESS)
+                        DEFAULT_UNINSTALL_ON_SUCCESS,
+                        DEFAULT_DIRECTORY_PATH)
```

Comment:
Convention in Cloudify code is to have the imports ordered by length. do move `DEFAULT_DIRECTORY_PATH` to one line above `DEFAULT_UNINSTALL_ON_SUCCESS`